build.sh

```diff
@@ -208,6 +208,19 @@ isMSBuildOnNETCoreSupported()
         elif [ "$__BuildOS" == "OSX" ]; then
             __isMSBuildOnNETCoreSupported=1
         fi
+    elif [ "$__BuildArch" == "arm" ] || [ "$__BuildArch" == "arm64" ] ; then
+        if [ "$__BuildOS" == "Linux" ]; then
+            if [ "$__DistroName" == "ubuntu" ]; then
+                __isMSBuildOnNETCoreSupported=1
```

Comment:
I concur - @leemgs please only keep the Distro/OS that have been tested for.

Currently, this check is also used as a gate to determine whether we should generate a nuget package for the product or not. Since they dont exist for the platforms/arch you are adding supported for, please confirm that it is not being generated.